include/swift/Basic/SimpleDisplay.h

```diff
@@ -50,6 +50,7 @@ namespace swift {
   HAS_TRIVIAL_DISPLAY(double);
   HAS_TRIVIAL_DISPLAY(bool);
   HAS_TRIVIAL_DISPLAY(std::string);
+  HAS_TRIVIAL_DISPLAY(const char *);
```

Comment:
We shouldn't need this, because we don't want bare `const char *`'s to show up anywhere in the evaluator.

Reply:
Uh, huh.

Reply:
Removed it.